Worksheet_B/README.md

```diff
@@ -0,0 +1,17 @@
+Print “ Welcome to Terminal 3 ”
+Print “ Password Required “
+Secret word = Pool
+Player strikes left: = 4
+Computer generates 14 words 4 letter words in random order
+Computer words = (“Pool, Roam, Seek, Dean, Foes, Half, Word, Copy, Cold, Fend, Wins, Pick, Died, Menu”)
```

Comment:
Could this be generalised, rather than hard-coding the secret word and the options?